docs/src/pages/premium-themes/onepirate/modules/components/AppBar.js

```diff
@@ -1,16 +1,8 @@
 import * as React from 'react';
-
-import { withStyles } from '@material-ui/styles';
 import MuiAppBar from '@material-ui/core/AppBar';
 
-const styles = (theme) => ({
```

Comment:
Was it intentional that we didn't add these styles?

Reply:
Yes, by default is white